src/firefly_exchange_client/constants.py

```diff
@@ -33,8 +33,18 @@
     "socketURL": "wss://dapi.api.arbitrum-prod.firefly.exchange",
     "webSocketURL": "wss://notifications.api.arbitrum-prod.firefly.exchange",
     "onboardingUrl": "https://trade-arb.firefly.exchange",
-    "UUID": "uuid-default"
+    "UUID": ""
   },
+  "MAINNET_ARBITRUM_INTERNAL": {
```

Comment:
Have we verified that attempting to access this works with a valid UUID and errors correctly with a non-valid UUID?

Reply:
we currently dont have any validity checks for UUID as of now, user can set it to anything. I think we should start saving this UUID somewhere in DB. Let me create a follow-up ticket for this.